dataset.proto

```diff
@@ -34,9 +44,11 @@ message DataSet {
     string url = 1;
   }
 
-  Categorization categorization = 1;
-  AdFraud        adfraud        = 2;
-  // 3 and 4 are reserved
-  Malicious      malicious      = 5;
-  Echo           echo           = 6;
+  Categorization         categorization = 1;
+  AdFraud                adfraud        = 2;
+  // 3 is reserved
+  Keyword                keyword        = 4;
```

Comment:
Might it be a good idea to indicate that either Keyword or Sentiment may have a value, but not both? Maybe it doesn't matter?